.docker/base-with-services/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# syntax=docker/dockerfile:1
+FROM base
+
+LABEL maintainer="AiiDAlab Team <aiidalab@materialscloud.org>"
+
+USER root
+WORKDIR /opt/
+
+ARG PGSQL_VERSION
+ARG RMQ_VERSION
+
+ENV PGSQL_VERSION=${PGSQL_VERSION}
+ENV RMQ_VERSION=${RMQ_VERSION}
+
+RUN mamba install --yes \
+     --channel conda-forge \
+     postgresql=${PGSQL_VERSION} && \
```

Comment:
postgresql is installed from conda forge to avoid the version may not exist from apt repo.